ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -135,19 +135,34 @@ def getStorageElementOptions( seName ):
 def getQueue( site, ce, queue ):
   """ Get parameters of the specified queue
   """
+  Tags = []
   grid = site.split( '.' )[0]
   result = gConfig.getOptionsDict( '/Resources/Sites/%s/%s/CEs/%s' % ( grid, site, ce ) )
   if not result['OK']:
     return result
   resultDict = result['Value']
+  ceTags = resultDict.get( 'Tag' )
+  if ceTags:
+    if isinstance( ceTags, basestring ):
+      Tags = fromChar( ceTags )
+    else:
```

Comment:
I think this is a redundant if/else: . With getOptionsDict you get all the options as strings by definition.